package.json

```diff
@@ -21,27 +21,37 @@
-    "test-integration": "mocha --timeout 600000 --reporter mocha-multi-reporters --reporter-options configFile=./test/mochareporters.json --compilers js:babel-core/register --require ./node_modules/babel-polyfill/dist/polyfill.min.js ./test/integration",
-    "test-unit": "mocha --reporter mocha-multi-reporters --reporter-options configFile=./test/mochareporters.json --compilers js:babel-core/register ./test/unit",
+    "test-e2e": "cross-env FORCE_NO_WRONG_FOLDER=true NODE_ENV=test RUNNING_IN_SPECTRON=true mocha --timeout 600000 --reporter mocha-multi-reporters --reporter-options configFile=./test/mochareporters.json ./test/e2e",
+    "test-integration": "cross-env NODE_ENV=test mocha --reporter mocha-multi-reporters --reporter-options configFile=./test/mochareporters.json ./test/integration",
+    "test-unit": "cross-env NODE_ENV=test mocha --reporter mocha-multi-reporters --reporter-options configFile=./test/mochareporters.json ./test/unit/",
     "lint": "eslint app test *.js",
-    "hot-server": "node -r ./node_modules/babel-register server.js --color",
-    "build-main": "cross-env NODE_ENV=production node -r ./node_modules/babel-register ./node_modules/webpack/bin/webpack --display=minimal --config webpack.config.electron.js --progress --profile --colors --color",
-    "build-renderer": "cross-env NODE_ENV=production node -r ./node_modules/babel-register ./node_modules/webpack/bin/webpack --display=minimal --config webpack.config.production.js --progress --profile --colors --color",
-    "build": "npm run build-main && npm run build-renderer",
-    "clean": "rimraf node_modules && npm install",
-    "start": "cross-env NODE_ENV=production electron ./",
-    "start-hot": "cross-env HOT=1 NODE_ENV=development electron -r ./node_modules/babel-register -r ./node_modules/babel-polyfill --inspect=5858 ./app/main/main.development --color",
-    "postinstall": "node node_modules/fbjs-scripts/node/check-dev-engines.js package.json && rimraf node_modules/jimp/browser",
-    "dev": "concurrently --kill-others \"npm run hot-server\" \"npm run start-hot\" --color",
+
+    "//": "PRODUCTION BUILD SCRIPTS",
+    "build-main": "cross-env NODE_ENV=production parcel build --target node --out-dir . --out-file main.js ./app/main/main.development.js",
+    "build-renderer": "cross-env NODE_ENV=production parcel build --out-dir dist --target electron --public-url . ./app/renderer/index.html",
+    "build": "npm run clean && npm run build-main && npm run build-renderer",
+    "start": "cross-env RUNNING_LOCALLY=true FORCE_NO_WRONG_FOLDER=true electron ./",
+
+    "//": "DEV BUILD SCRIPTS",
+    "watch-main": "cross-env NODE_ENV=developpment parcel --target node --out-dir . --out-file main.js ./app/main/main.development.js",
+    "watch-renderer": "cross-env NODE_ENV=developpment parcel --out-dir dist --target electron --public-url . ./app/renderer/index.html",
```

Comment:
^ These two scripts build the 'main' and 'renderer' thread and run a server that rebuilds on the fly.

`public-url` makes it relative instead of absolute so that `dist/` isn't treated as the root.